### ISSUE TITLE
PBP not responsive during install

### ISSUE THREAD

**Issue by squidds** [SvenKiljan/archlinuxarm-pbp#7]
I completed the initial install process to an SD card, booted the SD card, and then was completing the rsync step while attempting to finish the install on the EMMC. 

`[root@alarm digi]# mkdir -p /mnt/boot
[root@alarm digi]# mount /dev/mmcblk2p1 /mnt/boot

[root@alarm digi]# rsync -aAXq --exclude={"/dev/","/proc/","/sys/","/tmp/","/run/","/mnt/","/lost+found"} / /mnt
rsync: [sender] readlink_stat("/boot/dtbs/allwinner") failed: Input/output error (5)`

got this error, i was SSH'd to the pbp from my desktop, and there were errors in the tty on the pinebook. the error displayed in tty was "directory block failed checksum"

I tried to reboot to the SD card after the error, and the screen went black, have tried to hit the reset button on the PCB as well as flipping the EMMC switch. Right now just a black screen, unit is charged, no response to any input. I am not sure if for whatever reason the boot order is now EMMC even if there is an SD card inserted, but there should at least still be an LED displayed either way. I have played around with the old Arch (Nadia) on the pbp, have not seen an issue like this, but I do appreciate the effort on this and I know we can get it working for me. 

Thanks.

**Comment by SvenKiljan** [comment]
Seems like a bad SD card.

As noted by the [Pinebook Pro wiki](https://wiki.pine64.org/wiki/Pinebook_Pro#Bootable_Storage):
```
The boot order of the hard-coded ROM of its RK3399 SoC is: SPI NOR, eMMC, SD, USB OTG. 
```

If there is a (bad) boot record on the eMMC, the SoC will try to boot that. In your case, it probably hangs on this. One option you have is to buy the PINE USB eMMC reader, to write to the SoC using a different device. Specifically, erase the eMMC boot block with `dd if=/dev/zero of=/dev/sdX bs=1M count=32`. As long as the boot block (sectors 16-2048) is all zero, the eMMC will be skipped. You can also temporarily remove the eMMC card or disable it using the eMMC switch, and use a [good microSD card](https://wiki.pine64.org/wiki/Pinebook_Pro_Hardware_Accessory_Compatibility#microSD_Cards).

The power LED is controlled by the boot loader. If it fails to load for some reason, the power LED will not work.

**Comment by squidds** [comment]
Issue ended up being that the entire install was in fact borked. I wrote a Manjaro SD card and tried that and the system booted fine right after. I did too much on the OS while it was running from my SD card. I made an account and installed sudo and rsync had errors. I tried again after and only did a few things and was able to get your Arch running. But I have noticed that I was getting a few kernel panics and lock ups. May be my error.
